main.html

```diff
@@ -0,0 +1,7 @@
+<!DOCTYPE html>
+<html>
+<body>
+<h1 class="header">Hello Microverse</h1>
+
+</body>
+</html>
```

Comment:
:red_circle: **Please make sure your HTML file is renamed to index.html**
:red_circle: **Please make sure you have a `<head></head>` section  before the `<body>`.** 
:red_circle: **Please make sure your external CSS is linked to HTML using the <link href="path_to_css"> tag**